OpenUtau/App.axaml.cs

```diff
@@ -59,6 +60,11 @@ public class App : Application {
             if (resDict != null) {
                 Current.Resources.MergedDictionaries.Remove(resDict);
                 Current.Resources.MergedDictionaries.Add(resDict);
+                var langName = resDict.Source.OriginalString.Replace("/Strings/Strings.", "").Replace(".axaml", "");
```

Comment:
It is not meant to set the reference automatically. It is meant to follow system language if never manually set.

Reply:
OK never mind. This is not unreasonable either.

Reply:
Should I not save?
If I delete line 66, Language in the preference file will remain string.Empty.

Reply:
I wasn't sure if I should put it in Preferences.cs, but I couldn't write the code that references the resource dictionaries.